test/integration/crud/remove.test.js

```diff
@@ -107,6 +103,21 @@ describe('Remove', function () {
     }
   });
 
+  it('deleteOne', async () => {
```

Comment:
it's not clear from the description or the assertions what is under test here.  Also, why did we decide to add this test?  We have unified tests that cover all the assertions here (although implicitly)

Reply:
Agreed, [spec tests](https://github.com/mongodb/node-mongodb-native/blob/NODE-4684-rm-coll-insert-update-remove/test/spec/crud/unified/deleteMany-comment.json#L68) do check the "deletedCount" so no need for a dupe here.